### ISSUE TITLE
Redirect on main module can't be bundled

### ISSUE THREAD

**Issue by kitsonk** [denoland/deno#3082]
If the main URL for `deno bundle` points to a location that provides a redirect, you will get an error something like this:

```
$ deno bundle https://bit.ly/something bundle.js

error TS5012: Cannot read file 'https://bit.ly/something': Error: assert.

► 
error TS6053: File 'https://bit.ly/something' not found.

► 

Found 2 errors.
```

But this does not break `deno run https://bit.ly/something`.  (It also demonstrates that some errors in TypeScript don't have a location, and we should avoid printing out the `►`)

**Comment by Caesar2011** [comment]
> It also demonstrates that some errors in TypeScript don't have a location.

What do you mean by that?

**Comment by kitsonk** [comment]
We have logic for printing diagnostics that assumes all diagnostics have a code point. That logic is flawed.

**Comment by Caesar2011** [comment]
Oh, I understand. That is another issue...

**Comment by kitsonk** [comment]
This was resolved in Deno 1.5
